content/english/anti-vegan/humans-are-superior.md

```diff
@@ -0,0 +1,107 @@
+date: 2019-10-12T07:28:15+13:00
+name: tery-hardwicke
+contributors:
+    - 
+        name: tery-hardwicke
+        role: author
+    - 
+        name: sam-martin
+        role: editor
+draft: false
+description: "How to respond to the argument: We are entitled to use animals as a resource because humans are superior to nonhuman animals"
+weight: 5
+---
+# Argument
+
+> ...but, humans are superior...  
+>   
+> We are entitled to use animals as a resource because humans are superior to nonhuman animals
+
+# Rebuttal
+
+## Quick Responses
+
+> Superiority supposes an arbitrary line that falsely identifies one group as deservedly superior to another.   
+
+and  
+ 
+> Yes, humans are unique and distinct from other animals in our capacity for complex problem solving. However, pick any species and you will likely find an attribute that also makes them unique and distinct. We should celebrate difference and not deny moral consideration to those who are different.
+ 
+or 
+  
+> No unique set of abilities or characteristics, in itself, provides a criterion for ‘superiority’.  For humans to claim that humans are superior because we have specific unique attributes is just another form of prejudice. 
+ 
+or
+  
+> Even *if* it is the case we are uniquely superior, morality requires we care for those more vulnerable than ourselves; not that we treat them as a resource.
+
+besides
+
+> Do you *really* want to claim that when uniquely superior aliens discover our planet they will be morally entitled to use humans as a resource? 
+
+## Detail - Analytic
+
+### Humans are different to other animals
+
+That’s true. Humans have a capacity for complex problem solving that is not present in other animals. But why should a set of unique abilities or characteristics equate to superiority and give us the right to do what we like to those deemed inferior? Difference and uniqueness are not synonymous with superiority. 
+
+> Until recently it was considered that our capacity for abstract thought made us unique. However, “there is growing literature indicating that animals are capable of causal reasoning and abstraction”  
+> [Decision-Making From the Animal Perspective: Bridging Ecology and Subjective Cognition (2019)](https://www.frontiersin.org/articles/10.3389/fevo.2019.00164/full)  
+
+### On Superiority
+
+
+> One definition of 'Superior' is, "of higher rank, quality, or importance" - [Merriam-Webster](http://web.archive.org/web/20190711165845/https://www.merriam-webster.com/dictionary/superior)
+
+What most people forget (*or choose to overlook*) in using the term superior is that; *We alone* determine the ranking systems. *We alone* determine what denotes better quality. *We alone* determine what makes one thing more important than another. To presuppose that 'we' are superior to 'them' and to then *choose* an attribute that separates 'us' from 'them' as (somehow) demonstrating superiority is a fallacy. Identifying difference never identifies superiority.  
```

Comment:
Maybe add "even when it grants power of one over another" at the end? I really want to squeeze something in to ward off the might-makes-right crowd.

Reply:
K, did that.  Prob need to address 'might makes right...' at some point.   Mind you, are we sure it doesn't :)